exchanges/graphcache/src/types.ts

```diff
@@ -183,3 +183,8 @@ export interface StorageAdapter {
 }
 
 export type Dependencies = Record<string, true>;
+
+export const enum OperationType {
+  Read = 'READ',
+  Write = 'WRITE',
```

Comment:
I’d simply use a union type of `”read | “write”` here; we can keep it lowercase to keep it simple, because it’ll just be used directly and skip the use of enums.

Realistically we don’t need to use enums which transpile to objects and just stick to simple strings

Reply:
I like it!